src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx

```diff
@@ -182,6 +172,11 @@ const RawMemoRecyclerAssetList = React.memo(function RawRecyclerAssetList({
       refreshControl={disablePullDownToRefresh ? undefined : <RefreshControl />}
       renderAheadOffset={1000}
       rowRenderer={rowRenderer}
+      canChangeSize
+      layoutSize={{
+        height: deviceUtils.dimensions.height,
```

Comment:
Need to compute minimum height fort all the loading states here and use that as the initial layout size

Reply:
actually calculating the proper height was what slows this down. I'm assuming the blank 2-3 second screen is related to this list being for e.g. 12000px height when a user has quite a few NFTs. I think the "proper" fix for this would be to paginate nfts like originally planned. That way we don't need to render the full height of the list, and instead infinitely render as the user starts scrolling.